### PR TITLE
Fix #3411: Merge administrator_controls_download_permissions_view into single xml file.

### LINKED ISSUE #3411: Merge administrator_controls_download_permissions_view into single xml file.
Currently there are three versions of `administrator_controls_download_permissions_view.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by The-Anton**
## Explanation
Fixes #3411: Merged three ``` administrator_controls_download_permissions_view.xml ``` file (earlier created for mobile & tablet screens) into a single file. Variables for all the three different XML files are created and stored in ``` dimen.xml ```.

@rt4914  PTAL

Screenshots:

### Mobile-portrait

Before - 

After - 

### Mobile-landscape

Before - 

After - 

### Tablet-portrait

Before - 

After - 

### Tablet-landscape

Before - 

After - 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by The-Anton** [comment]
@rt4914 PTAL

**Comment by rt4914** [comment]
@The-Anton Apologies for delay. I will review it within next 24 hours. Meanwhile can you see that the last screenshot that you have added does not look like before screenshot, so can you please check that and fix it.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks,

**Comment by rt4914** [comment]
@The-Anton Can you just merge your PR with latest develop so that the CI checks can pass and we can merge it.

**Comment by The-Anton** [comment]
@rt4914 Thanks for reviewing it. I have updated the code for the tablet landscape screen so that there is no difference in the UI. The image in the PR description is also updated.

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml:9]: Please revert this change. It should not be part of this PR.

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
       </set>
     </option>
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/runConfigurations.xml:12]: Please revert this change. It should not be part of this PR.

File: build.gradle [hunk 3]
```diff
@@ -8,7 +8,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.2.2'
```
Comment (rt4914) [inline build.gradle]: Please revert this change. It should not be part of this PR.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@The-Anton PTAL. There are some unintentional changes. Please revert them. Thanks.

**Comment by rt4914** [comment]
> Hi, @The-Anton, the LGTM Label has been removed because the changes were requested on this PR. Thanks!.

@The-Anton Please do not resolve any conversation/comment threads. Instead you can reply on them and the person who has started those comment threads will resolve them. Thanks.

File: .idea/misc.xml (hunk 1 above)
Comment (The-Anton) [reply .idea/misc.xml:9]: Done.

File: .idea/runConfigurations.xml (hunk 2 above)
Comment (The-Anton) [reply .idea/runConfigurations.xml:12]: Done

File: build.gradle (hunk 3 above)
Comment (The-Anton) [reply build.gradle]: Done

**Comment by The-Anton** [comment]
Sorry, It was a mistake.
